superset-frontend/src/components/Select/OnPasteSelect.jsx

```diff
@@ -75,8 +75,8 @@ export default class OnPasteSelect extends React.Component {
   }
 
   render() {
-    const { selectWrap: SelectComponent, ...restProps } = this.props;
-    return <SelectComponent {...restProps} onPaste={this.onPaste} />;
+    const { selectWrap: SelectComponent, onPaste, ...restProps } = this.props;
+    return <SelectComponent {...restProps} onPaste={onPaste || this.onPaste} />;
```

Comment:
Can we revert the change here since it's not needed anymore?

Reply:
Done